setup.json

```diff
@@ -36,6 +36,7 @@
         "jinja2~=2.10",
         "jsonschema~=3.0",
         "kiwipy[rmq]~=0.7.4",
+        "importlib-metadata~=4.0",
```

Comment:
we probably want to up that to force people to upgrade to more performant versions

Reply:
Upped the pin to ~=4.3